README.md

```diff
@@ -4,7 +4,7 @@
 
 # Matt's DWM Auto Installing - Ricing - Script
 
-This script is meant for those who are intimidated by installing DWM by themselves or are (like Matt) just too lazy to do it. Just note that even after using this script there are still things you must do to get my setup. (Inlcuding moving scripts, setting up cronjobs and so on).
+This script is meant for those who are intimidated by installing DWM by themselves or are (like Matt) just too lazy to do it. Just note that even after using this script there are still things you must do to get my setup. (Including setting up cronjobs and so on).
```

Comment:
It seems like moving scripts is now handled [here](https://github.com/mtwb47/dwmricer/blob/master/dwminstall.sh#L25:L29)